docker-compose.yml

```diff
@@ -65,6 +64,7 @@ services:
       - ./custom-mods/:/custom-plugins/:ro
       - ./custom-maps/:/custom-maps/:rw
       - ./harmony-mods:/home/linuxgsm/serverfiles/HarmonyMods
+      - ./harmony-config:/home/linuxgsm/serverfiles/HarmonyConfig
```

Comment:
Can you create harmony-config directory similar to harmony-mods dir with a gitignore inside?

Reply:
Example https://github.com/samrocketman/docker-compose-lgsm-rust-dedicated-server/blob/main/harmony-mods/.gitignore